docs/content/quickstart/configurations.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "QuickStart: Configuration"
+descriptions: Control Porter's behavior with its config file
+layout: single
+---
+
+You now know how to install a bundle, passing in a credential or parameter set
+with Porter's default settings.
+Porter can be configured with some global settings, which are separate from any
+bundle specific configuration, and which apply across all bundles.
+This QuickStart walks you through how to set up and utilize the configuration
+file to achieve some common tasks.
+
+## Create configuration file
+
+First, let's create the configuration file. 
+
+In your PORTER_HOME directory, by default ~/.porter/, create a file with name
+config.yaml.
+```yaml
+touch ~/.porter/config.yaml
+```
+
+## Define default global namespace for bundles
+
+Instead of explicitly setting namespace for multiple bundles one by one, we can define a global default namespace
+in the configuration file like so:
+```yaml
+namespace = "dev"
+```
+
+## Enable debug logs by default
+
+Sometimes during development or debugging, it's helpful to get debug level logs
+until we have found the answer. You can deine debug log level in porter's configuration
+file like below to turn on debug logging as the default behavior:
+```yaml
+debug = true
+```
+
+## Work with bundle that contains sensitive data
+
+By default, porter does not persist sensitive data defined by users. It will
+return an error if a sensitive data is detected during a bundle execution. To
+try out porter with bundles that contain sensitive data, copy and paste below code into the
+configuration file:
+```yaml
+default-secrets-plugin = "filesystem"
+```
+
+This will enable the filesystem secret plugin to store any sensitive parameters
+provided by users, or sensitive outputs generated by bundles.
+However, this plugin should not be used in production environment since data
+will be stored as plain text in the local filesystem. Please use [azure-keyvault](/plugins/azure#secrets)
+or [hashicorp-vault](/plugins/hashicorp/#plugin-configuration) plugins for production.
```

Comment:
```suggestion
The filesystem secrets plugin should not be used in production since data
is stored as plain text in Porter's home directory. Please use either the [azure-keyvault](/plugins/azure#secrets)
or [hashicorp-vault](/plugins/hashicorp/#plugin-configuration) plugin in production.
```